Packs/ExpanseV2/Integrations/ExpanseV2/ExpanseV2.py

```diff
@@ -1505,6 +1724,35 @@ def manage_asset_tags_command(client: Client, args: Dict[str, Any]) -> CommandRe
     )
 
 
+def manage_asset_pocs_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    operation_type = args.get('operation_type')
+    if operation_type not in ASSET_POC_OPERATIONS:
+        raise ValueError(f'Operation type must be one of {",".join(ASSET_POC_OPERATIONS)}')
+
+    asset_type = args.get('asset_type')
+    if not asset_type or asset_type not in TAGGABLE_ASSET_TYPE_MAP:
+        raise ValueError(f'Asset type must be one of {",".join(TAGGABLE_ASSET_TYPE_MAP.keys())}')
+    mapped_asset_type = TAGGABLE_ASSET_TYPE_MAP[asset_type]
+
+    asset_id = args.get('asset_id')
+    if not asset_id:
+        raise ValueError('Asset id must be provided')
+
+    poc_ids = argToList(args.get('pocs'))
+    poc_emails = argToList(args.get('poc_emails'))
+
+    if len(poc_emails) > 0:
+        [poc_ids.append(p['id']) for p in client.list_pocs() if p['email'] in poc_emails]
+    pocs: List[str] = list(set(poc_ids))
+    if len(pocs) < 1:
+        raise ValueError('Must provide valid Point of Contact IDs or emails')
+
+    client.manage_asset_pocs(mapped_asset_type, operation_type, asset_id, pocs)
+    return CommandResults(
+        readable_output='Operation complete'
```

Comment:
I would suggest sending to this method the relevant title for each command and attach here. 
We should prove a better title here

Reply:
Good call, will add.